bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/channelhelper/EventCameraChannelHelper.java

```diff
@@ -24,21 +24,21 @@
 import org.openhab.core.types.State;
 
 /**
- * The {@link EventDoorbellChannelHelper} handles specific channels of doorbell events
+ * The {@link EventCameraChannelHelper} handles specific channels of doorbell events
  *
  * @author Gaël L'hopital - Initial contribution
  *
  */
 @NonNullByDefault
-public class EventDoorbellChannelHelper extends EventChannelHelper {
+public class EventCameraChannelHelper extends EventChannelHelper {
 
-    public EventDoorbellChannelHelper(Set<String> providedGroups) {
+    public EventCameraChannelHelper(Set<String> providedGroups) {
         super(providedGroups);
     }
 
     @Override
     protected @Nullable State internalGetHomeEvent(String channelId, @Nullable String groupId, HomeEvent event) {
-        if (groupId != null && GROUP_DOORBELL_SUB_EVENT.startsWith(groupId)) {
+        if (groupId != null && GROUP_SUB_EVENT.startsWith(groupId)) {
```

Comment:
I have a doubt: does it mean it will not be done for doorbell ?

Reply:
@Novanic @clinique : can one of you please confirm that this change is OK ?

Reply:
Hello @lolodomo , it is executed for doorbell as well. It is compared with "startsWith" and both constants (GROUP_SUB_EVENT and GROUP_DOORBELL_SUB_EVENT) are starting with "sub-event", see NetatmoBindingConstants.java. Therefore the change is fine.

I'm also using this binding versions for months with doorbell and presence cameras without problems. :-)

Reply:
Hmmm, @lolodomo could be right. It is compared with startsWith and both constants are starting with "sub-event". But the compare has the false order... But the behaviour is the same as before, I have to check what it causes. I'm using the binding with this change for months with doorbell and presence cameras ans have no problems.

Reply:
So I delay the merge until you have a clear status ?
By the way, you have please to make little fix in README, see my new review comments.

Reply:
Fixed and tested